compiler/cgen.nim

```diff
@@ -1117,7 +1125,10 @@ proc registerModuleToMain(g: BModuleList; m: PSym) =
   var
     init = m.getInitName
     datInit = m.getDatInitName
-  addf(g.mainModProcs, "N_LIB_PRIVATE N_NIMCALL(void, $1)(void);$N", [init])
+  if sfMainModule in m.flags:
+    addf(g.mainModProcs, "N_LIB_PRIVATE N_NIMCALL(static void, $1)(void);$N", [init])
```

Comment:
This ``static`` needs to be moved around.

Reply:
Ah, to "static N_LIB_PRIVATE N_NIMCALL(void, $1)(void);"?

Reply:
Yeah, but probably without the ``N_LIB_PRIVATE``.